### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: api/core/evaluate.js [hunk 1]
```diff
@@ -128,7 +128,8 @@ export const execute = async (
                 completed.add(id);
                 console.log(`Completed ${id}`);
               } catch (error) {
-                throw error;
+                // This should be reported via a note.
```
Comment (BarbourSmith) [inline api/core/evaluate.js:131]: What are notes?

File: es6/jsxcad-ui-app.js [hunk 2]
```diff
@@ -2860,6 +2860,36 @@ class EditNote extends ReactDOM$3.Component {
   }
 }
 
+class ErrorNote extends ReactDOM$3.Component {
```
Comment (BarbourSmith) [inline es6/jsxcad-ui-app.js:2863]: What do these look like in practice?

**Review by BarbourSmith — APPROVED** [review]
Does this look OK?

image

Other than that looks great to me 👍 👍

File: api/core/evaluate.js (hunk 1 above)
Comment (pentacular) [reply api/core/evaluate.js:131]: Ah, notes are the elements that notebooks are made of. :)

**Comment by pentacular** [comment]
> Does this look OK?
> 
> image
> Other than that looks great to me +1 +1

Thanks for catching that. :)

I forgot to uncomment something.

File: es6/jsxcad-ui-app.js (hunk 2 above)
Comment (pentacular) [reply es6/jsxcad-ui-app.js:2863]: Red text like "TypeError: Box(...).ez(...).clean(...).And(...).clip(...).flub is not a function"

**Pull request merged**